lib/vmdb/loggers/container_logger.rb

```diff
@@ -0,0 +1,53 @@
+module Vmdb::Loggers
+  class ContainerLogger < VMDBLogger
+    def initialize(logdev = STDOUT, *args)
+      super
+      self.level = DEBUG
+      self.formatter = Formatter.new
+    end
+
+    def level=(_new_level)
+      super(DEBUG) # We want everything written to the ContainerLogger written to STDOUT
+    end
+
+    def filename
+      "STDOUT"
+    end
+
+    class Formatter < VMDBLogger::Formatter
+      SEVERITY_MAP = {
+        "DEBUG"   => "debug",
+        "INFO"    => "info",
+        "WARN"    => "warning",
+        "ERROR"   => "err",
+        "FATAL"   => "crit",
+        "UNKNOWN" => "unknown"
+        # Others that don't match up: alert emerg notice trace
+      }.freeze
+
+      def call(severity, time, progname, msg)
+        # From https://github.com/ViaQ/elasticsearch-templates/releases Downloads asciidoc
+        {
+          :@timestamp => format_datetime(time),
+          :hostname   => hostname,
+          :level      => translate_error(severity),
+          :message    => prefix_task_id(msg2str(msg)),
+          :pid        => $PROCESS_ID,
+          :tid        => Thread.current.object_id,
```

Comment:
Sounds like I should make it a method on VMDBLogger::Formatter, although I have no idea how it is already `.to_s(16)` there...